### PR TITLE
[zh] sync 1.27 virtual-ips

### PR DESCRIPTION
**Pull request by Zhuzhenghao**
This PR resync page `content/zh-cn/docs/reference/networking/virtual-ips.md`.

### CODE REVIEW

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 1]
```diff
@@ -252,6 +252,45 @@ iptables:
...
+to `iptables-restore` operations, only making updates where Services or EndpointSlices have
+actually changed. This is a performance optimization.
+-->
+在 Kubernetes {{< skew currentVersion >}} 中，kube-proxy 默认采用最小方式进行 `iptables-restore` 操作，
+仅在 Service 或 EndpointSlice 实际发生变化的地方进行更新。这是一个性能优化。
+<!--
+The original implementation updated all the rules for all Services on every sync; this
+sometimes led to performance issues (update lag) in large clusters.
+-->
+最初的实现在每次同步时都会更新所有服务的所有规则；这有时会导致大型集群出现性能问题（更新延迟）。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:273]: Lines 262 - 273 shall be in a para. Do not separate them.

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 2]
```diff
@@ -252,6 +252,45 @@ iptables:
...
+removing that override and letting kube-proxy use the default value
+(`1s`) or at least a smaller value than you were using before upgrading.
+You can select the legacy behavior by disabling the `MinimizeIPTablesRestore`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+(you should not need to).
+-->
+如果你之前覆盖了 `minSyncPeriod`，你应该尝试删除该覆盖并让 kube-proxy 使用默认值（`1s`）或至少比升级前使用的值小。
+
+你可以通过禁用 `MinimizeIPTablesRestore`
+[特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)来选择旧行为（你应该不需要）。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:292]: Same issue. Lines 289 - 292 shall be in a para.

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 3]
```diff
@@ -562,6 +563,60 @@ IP addresses that are no longer used by any Services.
...
+对于 IPv6 用户可以使用等于或大于 /64 的掩码（以前是 /108）。
+
+<!--
+Users now will be able to inspect the IP addresses assigned to their Services, and
+Kubernetes extensions such as the [Gateway](https://gateway-api.sigs.k8s.io/) API, can use this new
+IPAddress object kind to enhance the Kubernetes networking capabilities, going beyond the limitations of
+the built-in Service API.
+-->
+用户现在能够检查分配给他们的服务的IP地址，Kubernetes扩展，如[网关](https://gateway-api.sigs.k8s.io/) API，可以使用这个新的IPAddress对象种类来增强Kubernetes的网络能力，超越内置服务API的限制。
+用户现在将能够检查分配给 Service 的 IP 地址，并且诸如[网关](https://gateway-api.sigs.k8s.io/) API 的 Kubernetes 扩展，可以使用这个新的 IPAddress 对象类型来增强 Kubernetes 网络功能，超越内置 Service API 的限制。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:601]: Line 600 - 601, 适当折行

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 4]
```diff
@@ -252,6 +252,42 @@ iptables:
...
+-->
+在 Kubernetes {{< skew currentVersion >}} 中，kube-proxy 默认采用最小方式进行 `iptables-restore` 操作，
+仅在 Service 或 EndpointSlice 实际发生变化的地方进行更新。这是一个性能优化。
+最初的实现在每次同步时都会更新所有服务的所有规则；这有时会导致大型集群出现性能问题（更新延迟）。
+
+<!--
+If you are not running kube-proxy from Kubernetes {{< skew currentVersion >}}, check
+the behavior and associated advice for the version that you are actually running.
+-->
+如果你没有从 Kubernetes {{< skew currentVersion >}} 运行 kube-proxy，请检查你实际运行的版本的行为和相关建议。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:277]:
```suggestion
如果你运行的不是 Kubernetes {{< skew currentVersion >}} 版本的 kube-proxy，
请检查你实际运行的版本的行为和相关建议。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 5]
```diff
@@ -252,6 +252,42 @@ iptables:
...
+If you were previously overriding `minSyncPeriod`, you should try
+removing that override and letting kube-proxy use the default value
+(`1s`) or at least a smaller value than you were using before upgrading.
+You can select the legacy behavior by disabling the `MinimizeIPTablesRestore`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+(you should not need to).
+-->
+如果你之前覆盖了 `minSyncPeriod`，你应该尝试删除该覆盖并让 kube-proxy 使用默认值（`1s`）或至少比升级前使用的值小。
+你可以通过禁用 `MinimizeIPTablesRestore`
+[特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)来选择旧行为（你应该不需要）。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:289]:
```suggestion
[特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)来选择执行旧的行为
（你应该不需要）。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 6]
```diff
@@ -562,6 +560,62 @@ IP addresses that are no longer used by any Services.
...
+如果你启用 `MultiCIDRServiceAllocator` [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gate/)
+和 [`networking.k8s.io/v1alpha1` API 组](/zh-cn/docs/tasks/administer-cluster/enable-disable-api/)，
+控制平面将用一个新的分配器替换现有的 etcd 分配器，使用 IPAddress 对象而不是内部的全局分配映射。
+与每个 Service 关联的 ClusterIP 地址将有一个对应的 IPAddress 对象。
+
+<!--
+The background controller is also replaced by a new one to handle the new IPAddress
+objects and the migration from the old allocator model.
+-->
+后台控制器也被一个新的控制器取代，来处理新的 IPAddress 对象和旧的分配器模型的迁移。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:581]:
```suggestion
后台控制器也被一个新的控制器取代，来处理新的 IPAddress 对象和从旧的分配器模型的迁移。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 7]
```diff
@@ -562,6 +560,62 @@ IP addresses that are no longer used by any Services.
...
+新分配器的主要好处之一是它取消了对 `service-cluster-ip-range` 的大小限制，对 IPv4 没有大小限制，
+对于 IPv6 用户可以使用等于或大于 /64 的掩码（以前是 /108）。
+
+<!--
+Users now will be able to inspect the IP addresses assigned to their Services, and
+Kubernetes extensions such as the [Gateway](https://gateway-api.sigs.k8s.io/) API, can use this new
+IPAddress object kind to enhance the Kubernetes networking capabilities, going beyond the limitations of
+the built-in Service API.
+-->
+用户现在能够检查分配给他们的服务的IP地址，Kubernetes扩展，如[网关](https://gateway-api.sigs.k8s.io/) API，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:597]:
```suggestion
用户现在能够检查分配给他们的 Service 的IP地址，Kubernetes扩展，如 [Gateway](https://gateway-api.sigs.k8s.io/) API，
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 8]
```diff
@@ -562,6 +560,62 @@ IP addresses that are no longer used by any Services.
...
+可以使用这个新的IPAddress对象种类来增强Kubernetes的网络能力，超越内置服务API的限制。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:598]:
```suggestion
可以使用这个新的 IPAddress 对象类别来增强 Kubernetes 的网络能力，解除内置 Service API 的限制。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 9]
```diff
@@ -562,6 +560,62 @@ IP addresses that are no longer used by any Services.
...
+用户现在将能够检查分配给 Service 的 IP 地址，并且诸如[网关](https://gateway-api.sigs.k8s.io/) API的
+Kubernetes 扩展，可以使用这个新的 IPAddress 对象类型来增强 Kubernetes 网络功能，超越内置 Service API 的限制。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:600]: delete these?

**Review by ydFu — APPROVED** [review]
/lgtm

**Pull request merged**